### PR TITLE
Fix/profilepic be fix

### PR DESCRIPTION
**Pull request by Ademola101**
This Pull Request is to fix the connection of the change profile picture by the user to the backend. The changed profile picture do not reflect currently on the FE.

After navigating the codebase and started the debugging process, I realize there was a problem with  response (image attached) sent from the server upon the PATCH request . It show that the image link generated by the backend is 'not found", thus there is no image to be displayed.

This suggests the issue is from the backend

Screenshot 2022-11-11 095214

### CODE REVIEW

**Comment by Ademola101** [comment]
This further commit fix the display issue when the user upload profile picture. The default Avatar is not updated 

This is what happen when user upload picture (BEFORE)
localhost_9000_workspace_636e91bc4746182adae9714c_plugin-chat_636e91c34746182adae9714f



After the fix


localhost_9000_workspace_636e91bc4746182adae9714c_plugin-chat_636e91c34746182adae9714f (1)

File: packages/main/src/components/protected/topbar/components/EditProfile.jsx [hunk 1]
```diff
@@ -149,6 +157,8 @@ const EditProfile = () => {
         });
       });
   };
+
+  console.log(userProfileImage);
```
Comment (JulianaSau) [inline packages/main/src/components/protected/topbar/components/EditProfile.jsx:161]: Remove this console.log from here

File: packages/main/src/components/protected/topbar/components/EditProfile.jsx [hunk 2]
```diff
@@ -356,11 +366,9 @@ const EditProfile = () => {
                     <div className="profile__img-wrapper">
                       <span className="pictureHeading">Profile photo</span>
                       <img
-                        ref={avatarRef}
+                        // ref={avatarRef}
```
Comment (JulianaSau) [inline packages/main/src/components/protected/topbar/components/EditProfile.jsx:369]: Remove this too

File: packages/main/src/components/protected/topbar/components/EditProfile.jsx [hunk 3]
```diff
@@ -49,29 +51,35 @@ const EditProfile = () => {
   const handleImageChange = event => {
     setState({ ...state, imageLoading: true });
     const [file] = event.target.files;
-
+    console.log(file);
```
Comment (JulianaSau) [inline packages/main/src/components/protected/topbar/components/EditProfile.jsx:54]: remove this code too

File: packages/main/src/components/protected/topbar/components/EditProfile.jsx [hunk 4]
```diff
@@ -49,29 +51,35 @@ const EditProfile = () => {
...
     if (file) {
       let fileReader = new FileReader();
-      fileReader.onload = function (event) {
-        avatarRef.current.src = event.target.result;
+      fileReader.onload = () => {
+        // if (avatarRef.current) {
+        //   avatarRef.current.src = fileReader.result;
+        // }
```
Comment (JulianaSau) [inline packages/main/src/components/protected/topbar/components/EditProfile.jsx:60]: Remove this too

File: packages/main/src/components/protected/topbar/components/EditProfile.jsx [hunk 5]
```diff
@@ -49,29 +51,35 @@ const EditProfile = () => {
...
+
+        setUserProfileImage(fileReader.result);
+        setimage(fileReader.result);
       };
 
       fileReader.readAsDataURL(file);
       const imageReader = file;
-
-      const formData = new FormData();
+      console.log(imageReader);
```
Comment (JulianaSau) [inline packages/main/src/components/protected/topbar/components/EditProfile.jsx:68]: Remove this code

File: packages/main/src/components/protected/topbar/components/EditProfile.jsx [hunk 6]
```diff
@@ -49,29 +51,35 @@ const EditProfile = () => {
...
+      let formData = new FormData();
       formData.append("image", imageReader);
       formData.append("height", 512);
       formData.append("width", 512);
-
+      console.log(formData.get("image"));
```
Comment (JulianaSau) [inline packages/main/src/components/protected/topbar/components/EditProfile.jsx:73]: Remove this too

File: packages/main/src/components/protected/topbar/components/EditProfile.jsx [hunk 7]
```diff
@@ -49,29 +51,35 @@ const EditProfile = () => {
...
       authAxios
         .patch(
           `/organizations/${orgId}/members/${user._id}/photo/upload`,
           formData
         )
         .then(res => {
+          console.log(res);
           const newUploadedImage = res.data.data;
-          setUserProfileImage(newUploadedImage);
+          // setUserProfileImage(newUploadedImage);
```
Comment (JulianaSau) [inline packages/main/src/components/protected/topbar/components/EditProfile.jsx:82]: remove this too

**Review by JulianaSau — CHANGES_REQUESTED** [review]
Remove all console.logs from your code and any commented out code

**Review by JulianaSau — APPROVED** [review]

**Pull request merged**